src/Bridges/vaftosafbridge.jl

```diff
@@ -0,0 +1,148 @@
+const VectorToScalarBridgeSets = Union{MOI.Zeros,
+                                       MOI.Nonpositives,
+                                       MOI.Nonnegatives}
+get_scalar_set(T, ::Type{MOI.Zeros}) = MOI.EqualTo{T}
+get_scalar_set(T, ::Type{MOI.Nonpositives}) = MOI.LessThan{T}
+get_scalar_set(T, ::Type{MOI.Nonnegatives}) = MOI.GreaterThan{T}
+
+get_vector_set(::Type{MOI.EqualTo{T}}) where T = MOI.Zeros
+get_vector_set(::Type{MOI.LessThan{T}}) where T = MOI.Nonpositives
+get_vector_set(::Type{MOI.GreaterThan{T}}) where T = MOI.Nonnegatives
+
+"""
+    VectorToScalarBridge{T, S}
+
+The `VectorToScalarBridge` splits an `VectorAffineFunction` into a series of
+`ScalarAffineFunction`s.
+
+Supported sets are:
+ - `MOI.Zeros` =>`MOI.EqualTo{T}`
+ - `MOI.Nonnegatives` =>`MOI.GreaterThan{T}`
+ - `MOI.Nonpositives` =>`MOI.LessThan{T}`
+"""
+struct VectorToScalarBridge{T, S} <: AbstractBridge
```

Comment:
Would be nice to have
```julia
struct VectorToScalarBridge{T, F, S} <: AbstractBridge
    constriants::Vector{CI{F, S}}
end
```
where `F` can also be, e.g. `SingleVariable` and `ScalarQuadraticFunction`.
You can leave it for future work as the tools needed are being added in https://github.com/JuliaOpt/MathOptInterface.jl/pull/455